commit 23d26f9c9fd94406d0c1f4365e6180d59b744861
Author: Scott Mayhew <smayhew@redhat.com>
Date:   Mon Nov 2 08:45:09 2015 -0500

    exportfs: Restore the EAI_NONAME check in host_pton()
    
    Commit d89e3fc7 removed the EAI_NONAME check altogether instead of just
    moving the NULL check.  This causes exportfs -u to incorrectly exit
    with 1 whenever there's more than one MCL_FQDN export in the exportlist.
    
    Signed-off-by: Scott Mayhew <smayhew@redhat.com>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/support/export/hostname.c b/support/export/hostname.c
index 169baa5..7a44d42 100644
--- a/support/export/hostname.c
+++ b/support/export/hostname.c
@@ -134,6 +134,8 @@ host_pton(const char *paddr)
 			break;
 		}
 		return ai;
+	case EAI_NONAME:
+		break;
 	case EAI_SYSTEM:
 		xlog(D_GENERAL, "%s: failed to convert %s: (%d) %m",
 				__func__, paddr, errno);
